src/pages/Leaderboard.css

```diff
@@ -50,3 +50,10 @@
 #leaderboard-avatar {
   padding: 5% 0%;
 }
+
+#currentStudent-name{
```

Comment:
Follow the naming convention for ids for everywhere like this #page-name-element-appropriate-name